Library/.rubocop.yml

```diff
@@ -8,6 +8,19 @@ FormulaAudit:
 FormulaAuditStrict:
   Enabled: true
 
+# enable all pending rubocops
+AllCops:
+  NewCops: enable
+
+# don't allow cops to be disabled in formulae
+Style/DisableCopsWithinSourceCodeDirective:
+  Enabled: true
+  Exclude:
+  # TODO: really long lines but hard to resolve (but would be nice to do).
+  - '**/Formula/libgraphqlparser.rb'
```

Comment:
We could convert [those long](https://github.com/Homebrew/homebrew-core/blob/0cdca414f2f56f3db70e4990937d13c2ca63b053/Formula/libgraphqlparser.rb#L33-L60) lines to json:

```json
{
  "kind": "Document",
  "loc": {
    "start": {
      "line": 1,
      "column": 1
    },
    "end": {
      "line": 1,
      "column": 9
    }
  },
  "definitions": [
    {
      "kind": "OperationDefinition",
      "loc": {
        "start": {
          "line": 1,
          "column": 1
        },
        "end": {
          "line": 1,
          "column": 9
        }
      },
      "operation": "query",
      "name": null,
      "variableDefinitions": null,
      "directives": null,
      "selectionSet": {
        "kind": "SelectionSet",
        "loc": {
          "start": {
            "line": 1,
            "column": 1
          },
          "end": {
            "line": 1,
            "column": 9
          }
        },
        "selections": [
          {
            "kind": "Field",
            "loc": {
              "start": {
                "line": 1,
                "column": 3
              },
              "end": {
                "line": 1,
                "column": 7
              }
            },
            "alias": null,
            "name": {
              "kind": "Name",
              "loc": {
                "start": {
                  "line": 1,
                  "column": 3
                },
                "end": {
                  "line": 1,
                  "column": 7
                }
              },
              "value": "user"
            },
            "arguments": null,
            "directives": null,
            "selectionSet": null
          }
        ]
      }
    }
  ]
}
```

If it's ok, I'll create a PR

Reply:
> If it's ok, I'll create a PR

Yes, please do ❤️

Reply:
https://github.com/Homebrew/homebrew-core/pull/53259